### PR TITLE
fix: DEV-3672: I can't reassign label to bounding box with a hotkey

### PR DESCRIPTION
**Pull request by yyassi-heartex**

### CODE REVIEW

File: src/tags/control/Label.js [hunk 1]
```diff
@@ -236,12 +236,18 @@ const Model = types.model({
...
+    return self.onLabelInteract();
+  },
+
+  onClick() {
+    return self.onLabelInteract();
+  },
+
+  onLabelInteract() {
+    if(!self.annotation.isDrawing) {
+      self.toggleSelected();
```
Comment (bmartel) [inline src/tags/control/Label.js:248]: what is the return of `toggleSelected`? I seen this was previously returning the result of that function in `onHotKey`. Just want to make sure this is returning the equivalence as previous in all scenarios.

Comment (yyassi-heartex) [reply src/tags/control/Label.js:248]: appears to return nothing based on what I see here https://github.com/heartexlabs/label-studio-frontend/blob/0aafae54f03808dddbce54642c1aac281ea487d4/src/tags/control/Label.js#L114

Comment (yyassi-heartex) [reply src/tags/control/Label.js:248]: I opted to return false as the onClick was returning false

**Review by hlomzik — APPROVED** [review]

**Pull request merged**